src/OrchardCore.Modules/OrchardCore.OpenApi/OrchardApiDefinition.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Mvc.ApiExplorer;
+using Microsoft.OpenApi.Models;
+using OrchardCore.OpenApi;
+
+namespace OrchardCore.Swagger
+{
+    public class OrchardApiDefinition : IOpenApiDefinition
+    {
+        public string Name => "OrchardCoreAPI";
+
+        public OpenApiDocument Document => new OpenApiDocument()
+        {
+            Info = new OpenApiInfo()
+            {
+                Version = "v2",
+                Title = "OrchardCore API",
+                Description = "An API to manage the OrchardCore installation",
+                Contact = new OpenApiContact
+                {
+                    Name = "Orchard Team",
+                    Email = "info@orchardproject.net",
+                    Url = new Uri("https://www.orchardproject.net")
+                }
+            }
+        };
+
+        public Func<string, ApiDescription, bool> ApiDescriptionFilterPredicate => (name, description) =>
```

Comment:
These should be moved in to the modules to which they belong.

Reply:
@Jetski5822 I consider all OrchardCore API's one API.
I am quite happy to implement it like that, but it would mean there are many API documents.